### PR TITLE
liveness initialdelay: must be greater than readiness initialdelay

### PR DESCRIPTION
**Pull request by hervenicol**
# TLDR
Liveness initialdelay: must be greater than readiness initialdelay to avoid crashloop at startup.

# Details
We don't much care about having a low readiness initialdelay: the risk is to have an invalid pod answering errors to requests, during startup phase.
On top of that, the "memberlist" servicediscovery mechanism (based on a headless service) requires a low readiness initialdelay, and setting it too high breaks discovery.

On the other hand, liveness initialdelay has to be high to avoid killing services during fetching of data for bigger Loki clusters.

Towards https://github.com/giantswarm/roadmap/issues/718

### CODE REVIEW

**Comment by hervenicol** [comment]
ping @giantswarm/team-atlas
PTAL, merging it would help for next tests / fixes ;)

**Review by TheoBrigitte — APPROVED** [review]
LGTM, when tested.
